Dockerfile

```diff
@@ -25,6 +27,16 @@ ADD laminas-ci.schema.json /action/
 COPY --from=compiler /usr/local/source/dist/main.js /action/
 RUN chmod u+x /action/main.js
 
+# Setup PHP
+RUN mkdir -p /usr/local/bin /usr/local/etc /usr/local/lib
+COPY --from=php /usr/local/bin/php /usr/local/bin
+COPY --from=php /usr/local/etc/* /usr/local/etc
+COPY --from=php /usr/local/lib/php/* /usr/local/lib/php
+COPY --from=php /usr/lib/* /usr/lib
+COPY --from=php /lib/* /lib
```

Comment:
it's these parts that worry me mixing the languages. if they both rely on a linked lib in different versions for instance.

Reply:
Similar thoughts: would prefer a `composer install` happening with our base image, but we can improve on that later

Reply:
There is no need for `composer install`, thats actually to execute the `composer-semver.phar` I am building with https://github.com/boesing/composer-semver-docker

But the more I think about this, the more it annoys me that it has to be that hacky...